src/test/java/org/opentripplanner/transit/raptor/api/transit/CostCalculatorTest.java

```diff
@@ -30,15 +35,15 @@ public void transitArrivalCost() {
         assertEquals(1000, prev.cost());
 
         // Simulate round 1
-        assertEquals("Board cost", 500, subject.transitArrivalCost(true, fromStop, 0, 0, 0));
-        assertEquals("Board + transit cost", 600, subject.transitArrivalCost(true, fromStop, 0, 1, 0));
-        assertEquals("Board + transit + stop cost", 625, subject.transitArrivalCost(true, fromStop, 0, 1, 1));
-        assertEquals("Board cost", 550, subject.transitArrivalCost(true, fromStop, 1, 0, 0));
+        //assertEquals("Board cost", 500, subject.transitArrivalCost(true, fromStop, 0, 0, 0, TRANSIT_RELUCTANCE_1));
+        //assertEquals("Board + transit cost", 600, subject.transitArrivalCost(true, fromStop, 0, 1, 0, TRANSIT_RELUCTANCE_1));
+        assertEquals("Board + transit + stop cost", 625, subject.transitArrivalCost(true, fromStop, 0, 1, TRANSIT_RELUCTANCE_1, 1));
+        assertEquals("Board cost", 550, subject.transitArrivalCost(true, fromStop, 1, 0, TRANSIT_RELUCTANCE_1, 0));
```

Comment:
Why are the first two assertions commented? If not needed anymore, could the lines be removed?

Reply:
Thank you for spotting this one - I have commented in the test lines again.